### DISCUSSION TITLE
Hyprland-like Gradient Borders for Wayland

### DISCUSSION THREAD

**Discussion by VehementHam** [qtile/qtile#4639] [Ideas]
I want Qtile to have an option to enable gradient borders when it is being run as a Wayland compositor. Here is an example of gradient borders: https://lemmy.ml/post/7194344

**Comment by elParaguayo** [comment]
I've done this in x11 but just to see if it could be done. I suspect it's not too hard in Wayland. 

The real issue I have is how we make something like that configurable.
